.circleci/test-content-server.sh

```diff
@@ -46,21 +46,23 @@ if grep -e "$MODULE" -e 'all' $DIR/../packages/test.list; then
 
   cd ../../
   npx pm2 delete servers.json && npx pm2 start servers.json
-  # ensure email-service is ready
-  check 127.0.0.1:8001/__heartbeat__
   cd packages/fxa-content-server
   mozdownload --version 58.0 --destination firefox.tar.bz2
 
   if [ -n "${PAIRING}" ]; then
     wget https://s3-us-west-2.amazonaws.com/fxa-dev-bucket/fenix-pair/desktop/7f10c7614e9fa46-target.tar.bz2
     mozinstall 7f10c7614e9fa46-target.tar.bz2
+    # ensure email-service is ready
+    check 127.0.0.1:8001/__heartbeat__
```

Comment:
Moved the `check` here until after `mozinstall` has completed. I noticed that `check` would often take 40-60 seconds, whereas `mozinstall` seems to take at least 30. So, seemed to make sense to spend some time installing rather than just waiting in a loop.